appveyor.yml

```diff
@@ -28,18 +28,25 @@ build_script:
   - 7z a multi-geth-win64.zip .\build\bin\geth.exe
   - ps: Get-FileHash multi-geth-win64.zip -Algorithm SHA256
   - ps: Get-FileHash multi-geth-win64.zip -Algorithm SHA256 | Out-File multi-geth-win64.zip.sha256
+  - 7z a multi-geth-alltools-win64.zip .\build\bin\*
```

Comment:
@sorpaas BTW, it occurred to me that we could use the `%GETH_ARCH%` (as above, currently yielding only `amd64`) to use a build-specific architecture variable in the name, something like `multi-geth-windows-amd64.zip`. This would keep options open for addition of extra build archs if that ever were to happen. Not sure worth tackling now, but just a heads up.